.github/workflows/release.yml

```diff
@@ -1,15 +1,22 @@
 name: Release
 
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
 on:
   push:
     branches:
-      - alpha
-      - beta
       - main
+      - alpha
 
 jobs:
+  build:
+    uses: ar-io/ar-io-sdk/.github/workflows/build.yml@alpha
```

Comment:
is main intended to build from `@alpha` ?

Reply:
good catch - should be `@main`

Reply:
well the job runs from each branch. shouldn't have to be specific

Reply:
[eb7036b](https://github.com/ardriveapp/turbo-sdk/pull/83/commits/eb7036b747e33ebaf803d91798beecb306f65d24)

Reply:
I'd expect `alpha` release to wait for the build action on `alpha`, while `main` release to wait for build action on `main`. I think this would now have the `alpha` release wait on the build action of `main`

Reply:
it's just referencing the workflow file - not the actual execution of it

Reply:
i.e. it will trigger the build file on the `main` branch